### DISCUSSION TITLE
Indication about the date when the release has been created

### DISCUSSION THREAD

**Discussion by mouradchama** [GitTools/GitVersion#3636] [Ideas]
**Is your improvement request related to a problem? Please describe.**

I'm using Gitversion in my projects and it works perfectly fine. Recently I received a request to add an information about the release date in the version number.

Now, I'm wondering how I can mix SemVer with the release date, something like 1.2.0-R2307

## Detailed Description & Context
For the communication pupose, with customers or the other departments (support, customers, management..) we would like to add an indication about the date when the release has been created, we are using year/month to manage releases.
I don't know if this request has been addressed before but I would like to know if you have any suggestion to achieve what I want.

## Possible Implementation
My first idea is to use the Tag when I generate from the Main/Master branch : 1.2.0-R2307 :
1. Use a fix value : R2307 in the Tag property, the problem is that I have to change it each month, each release (we are releasing each month)
2. Use dynamic tag (I don't know if it's feasable), even if it's not faisable I can do it out of the box (I can use the build date time in Azure Devops), the problem is if I provide a hotfix the month after my release will be 1.2.1-R2308, which is soething that I would like to avoid.

Thank you in advance for any suggestion or advice

Regards,

**Comment by HHobeck** [comment]
Hi there. I'm moving this to the discussion section.

**Comment by HHobeck** [comment]
I'm actually not sure if it is a good idea to use the year and month value as a pre-release number only. But if you like you can do it and use the last commit date for e.g. `AssemblySemVer`:

```yaml
assembly-versioning-format: '{Major}.{Minor}.{Patch}-R{CommitDate}'
commit-date-format: yyMM
```
